lib/testing/src/lib/process-services/pages/process-list.page.ts

```diff
@@ -45,4 +45,8 @@ export class ProcessListPage {
     checkContentIsNotDisplayedByColumn(column: string, processName: string): Promise<void> {
         return this.dataTable.checkContentIsNotDisplayed(column, processName);
     }
+
+    selectRow(processName): Promise<void> {
```

Comment:
if selectRow doesn't return anything you can declare selectRow async and wait on this.dataTable.selectRow

Reply:
Changed it to:
```
    async selectRow(processName): Promise<void> {
        return await this.dataTable.selectRow('Name', processName);
    }
```